build/release.sh

```diff
@@ -44,6 +78,14 @@ increment_version() {
     # Old version
     OLD_VERSION=$(./build/current_version.sh)
 
+    # The master branch should not match the previous release tag
```

Comment:
This isn't necessarily the master branch right? If we're coming from a patch this would be some detached state.